source/fe/mapping_fe.cc

```diff
@@ -909,12 +909,82 @@ MappingFE<dim, spacedim>::transform_real_to_unit_cell(
   const typename Triangulation<dim, spacedim>::cell_iterator &cell,
   const Point<spacedim> &                                     p) const
 {
-  Assert(false, StandardExceptions::ExcNotImplemented());
+  const std::vector<Point<spacedim>> support_points =
+    this->compute_mapping_support_points(cell);
+
+  Tensor<1, spacedim> DF[dim];
+  // [TODO]
+  // When 2nd derivatives are implemented, we'll uncomment this
+  // Tensor<1, spacedim> D2F[dim][dim];
+
+  Point<dim>      p_unit;
+  Point<spacedim> mapped_point;
+  Point<spacedim> p_minus_F;
+
+  Point<dim>     f;
+  Tensor<2, dim> df;
+
+  const double       eps        = 1.e-12 * cell->diameter();
+  const unsigned int loop_limit = 10;
+
+  unsigned int loop = 0;
+  do
+    {
+      auto mapped_point = support_points[0] * 0.0;
```

Comment:
?? Why not be explicit:
```suggestion
      Point<spacedim> mapped_point;
```